<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBluetooth</class>
 <widget class="QDialog" name="DialogBluetooth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conex√£o bluetooth</string>
  </property>
  <widget class="QComboBox" name="cbOptions">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dispositivo:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Conectar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
